src/rpc/server/scala/serv/src/main/scala/com/ctc/omega_edit/grpc/Editors.scala

```diff
@@ -106,12 +107,12 @@ class Editors extends Actor with ActorLogging {
     case Find(id) =>
       sender() ! context.child(id)
 
-    case DestroyActor(id) =>
+    case DestroyActor(id, t) =>
       context.child(id) match {
         case None => sender() ! Err(Status.NOT_FOUND)
         case Some(s) =>
-          context.system stop s
```

Comment:
@jw3 Thank you for the help. This was actually change I had made recently because before we were point `s ! PoisonPill`. However, this was causing issues with sessions not being destroyed and this line fixed that issue but introduced the other we saw. Thanks again for the help!

Reply:
:+1: